plc4x/throwing-the-towel.adoc

```diff
@@ -0,0 +1,227 @@
+
+I will continue to invest my free time into open-source. However, just in projects that I believe have a sustainable future or just for fun for other projects which I have some other strange form of affection for.
+
+Even if I have completely lost my faith in open-source in the OT-industry, I still believe in open-source in general.
+
+=== Glimpse into the future
+
+There's something big lurking around the corner, that might become an issue soon and which could speed up some things.
+
+Once the CRA and PLD initiatives are becoming binding legal frameworks in europe and probably also similar initiatives elsewhere, I expect the `house-keeping` in projects such as Apache PLC4X to skyrocket.
+
+If there's nobody willing or able to do that, that's going to create facts a lot quicker than most people will expect.
+
+Even if money would definitely be well invested in supporting open-source projects in general.
+Especially those projects that don't have any form of commercial backing are probably most in need of funding, as they will not have the power to prepare for these legal changes.
+Some rules for some projects will most probably require them to execute external code audits.
+Which group of private people will be willing to invest the significant amounts of their private money in order to pay for something, just to be able to continue giving away something for free?
+I see no way around significant investments by entities relying on open-source but also legal bodies such as the European Union in order to keep the lights on in these projects.
+
+One thing people relying on open-source should generally consider: Using an open-source project in the EU which is hosted at an organisation which is considered an  `open-source steward` will reduce the amount of hoops they definitely will need to jump through in order to sell their products significantly once CRA and PLD are in place.
+The ASF is such a place.
+
+Using projects that are not maintained by an open-source steward might become impossible, as these would either need to apply to all rules or using them will become illegal if the company using it doesn't take care of this (That's a hell of a lot of rules).
+Apache PLC4X is, as far as I know, the only open source project for communication with various types of industrial equipment, which is also hosted by an organization considered an open-source steward.
+The only other project in this sector I know of would be the Eclipse Milo project, as the Eclipse foundation is also considered an open-source steward.
+Having a look at most other open-source projects in that field.
+Most are un-paid single person projects.
+I see no chance of being able to continue to use these in a CRA and PLD regulated world.
+NodeRED is going to be interesting, but I am not a lawyer and I can't say what's going to happen there.
```

Comment:
NodeRED .... macht das nicht eigentlich nur die Orchestration bzw. Ablaufgeschichte. Hat das Konnektoren im Core?

Was willst du mit "but I am not a lawyer and I can't say what's going to happen there" sagen?